app/controllers/api/account.php

```diff
@@ -1500,25 +1500,27 @@
         $response->dynamic($user, Response::MODEL_USER);
     });
 
-App::delete('/v1/account')
-    ->desc('Delete Account')
+App::patch('/v1/account')
+    ->desc('Update Status (Block)')
     ->groups(['api', 'account'])
-    ->label('event', 'account.delete')
+    ->label('event', 'account.updateStatus')
     ->label('scope', 'account')
     ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_JWT])
     ->label('sdk.namespace', 'account')
-    ->label('sdk.method', 'delete')
-    ->label('sdk.description', '/docs/references/account/delete.md')
-    ->label('sdk.response.code', Response::STATUS_CODE_NOCONTENT)
-    ->label('sdk.response.model', Response::MODEL_NONE)
+    ->label('sdk.method', 'block')
+    ->label('sdk.description', '/docs/references/account/block.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_USER)
+    ->param('block', null, new Boolean(), 'Is account blocked?')
     ->inject('request')
     ->inject('response')
     ->inject('user')
     ->inject('dbForProject')
     ->inject('audits')
     ->inject('events')
     ->inject('usage')
-    ->action(function ($request, $response, $user, $dbForProject, $audits, $events, $usage) {
+    ->action(function ($block, $request, $response, $user, $dbForProject, $audits, $events, $usage) {
```

Comment:
Not sure what to do with the parameters here. Since this method doesn't really need any params. 
Might be a good idea to remove the params altogether.